infrastructure/variables.tf

```diff
@@ -134,6 +134,20 @@ variable "reports_cron" {
 
 # endregion
 
+# region incomplete envelopes monitoring
+
+variable "incomplete_envelopes_cron" {
+  default     = "0 0 7 ? * MON-FRI"
```

Comment:
may be we should run it after 9 ?

Reply:
as commented in config - up for discussion. but 9am sounds good to me. will change